Safety/Src/PWM.c

```diff
@@ -99,9 +105,10 @@ void HAL_TIM_IC_CaptureCallback(TIM_HandleTypeDef *htim) {
         uint16_t time_diff = HAL_TIM_ReadCapturedValue(htim, TIM_CHANNEL_1);
 
         if (ppm_index != 0) { // the first edge doesn't give us any data
-            capture_value[ppm_index - 1] = time_diff - PPM_OFFSET;
-        }
-        ppm_index = (ppm_index + 1) % (PPM_NUM_CHANNELS + 1); // index should reset, but just in case
+            capture_value[ppm_index - 1] = (time_diff - PPM_OFFSET) / 2.5; //Divide by 5 to keep PPM and PWM at same scale
```

Comment:
Why do they need to be at the same scale?

Reply:
It makes a lot more sense for them to be at the same scale. That means I can just call PWM_SetAll(PPM_Get()) and it works. You lose some accuracy, but that accuracy is irrelevant anyways since it is mostly being used for outputting PWM